storage.rst

```diff
@@ -4,36 +4,64 @@
+   This behavior improves reliability and does not require manual intervention. However, users can still manage storage settings from the web interface.  
 
-In both cases, the partition will be formatted using the ext4 filesystem and mounted as storage at ``/mnt/data``.
+Persistent log storage can be disabled (not recommended), or moved to a different disk if needed.  
+If disabled, the system will automatically reconfigure it during a future upgrade.
 
-The system will then be reconfigured as follows:
+Manual configuration
+^^^^^^^^^^^^^^^^^^^^
 
-- rsyslog will write logs also inside ``/mnt/data/logs/messages`` file
-- logrotate will rotate ``/mnt/data/logs/messages`` once a week
+Manual configuration of additional storage is still available and works as follows:
 
-Extra generated data like metrics, will be synced from in-memory filesystem to persistent storage once a day during the night.
-To remove the data storage and restore in-memory log retention only, click on button :guilabel:`Remove storage`.
+* If using an additional device, connect it to the system.
+* Access the ``Storage`` page under the ``System`` section in the right-hand menu.
+* Select the storage device where logs should be saved.
+* Click on the button :guilabel:`Format and configure storage`.
+* If the selected device is the **primary disk**, the system will generate a new partition using any unallocated space.
+* If an **additional disk** is selected, the system will erase all existing partitions and data and create a single new partition.
+
+The storage is then:
+
+* Formatted with the ``ext4`` filesystem
+* Mounted at ``/mnt/data``
+* Used by ``rsyslog`` to write logs to ``/mnt/data/log/messages``
+* Rotated weekly by ``logrotate``
+* Synced daily (at night) for additional data like metrics
+
+To remove persistent storage and return to in-memory logging, click on the button :guilabel:`Remove storage`.
+
+Virtual Machines
+----------------
+When installing NethSecurity on a virtual machine, the recommended method is to generate the virtual disk from the official image.
+In this mode, logs are not stored persistently by default.
+To enable persistent log storage, you must attach a second virtual disk to the virtual machine.
+The automatic internal partitioning method is not supported on virtual disks.
+
+Behavior in versions prior to 8.6
+---------------------------------
+
+In earlier versions of NethSecurity, logs were written by default to a **volatile in-memory directory** to avoid stressing the root filesystem in case of failure.
+
+* Memory-based storage had **limited capacity**
+* Old logs were automatically removed as space filled up
+* **All logs were lost** after a reboot or system failure
+
+To persist logs, storage had to be configured **manually**, either by using unallocated space on the system disk or by attaching a secondary disk.  
+This step was necessary to avoid log loss, but required explicit action by the user.
```

Comment:
System is still more reliable without a persistent log, but we are enabling it to ease the life of the support team.
So it's better to just describe the old behavior without more explanation.

```suggestion
In earlier versions of NethSecurity, logs were written by default to a **volatile in-memory directory**.
To persist logs, storage had to be configured **manually**, either by using unallocated space on the system disk or by attaching a secondary disk.  
```